src/components/EventList.tsx

```diff
@@ -59,10 +63,21 @@ const EventList = ({ events, org } : EventListProps) : JSX.Element => {
                             />
                         </View>
                         <View data-test="location-title">{ e.location.title }</View>
-                        <Button data-test="sign-up-button" marginTop="size-50" variant="cta">
+                        <Button
+                            data-test="sign-up-button"
+                            marginTop="size-50"
+                            onPress={ () => putEventResponse(e.id, org!.id) }
```

Comment:
Same here as above, i.e. we shouldn't use non-null assertion unless it's inevitable to type something as `undefined` or `null` (or `any`/`unknown`), even though we know it never can be.